tests/GraphQL.Integration.Tests/QueryAndMutationTests/Base.cs

```diff
@@ -180,6 +180,9 @@ public async void PreprocessHttpRequestMessageIsCalled()
             };
 
             var defaultHeaders = StarWarsClient.HttpClient.DefaultRequestHeaders;
+            var userAgentOption = StarWarsClient.Options.DefaultUserAgentRequestHeader;
```

Comment:
Why are you doing this here? This does not test the new functionality!

Please create a new test case to prove that the user agent header is correctly modified.

Reply:
@KirovAir I resolved merge conflicts and reverted changes in this test file. As @rose-a said please add new test.

Reply:
Sorry guys I completely overlooked it. Currently on holiday until next thursday. Let me resolve it when I get back home.

Reply:
@KirovAir Let's finish it and I'm glad to merge.